### ISSUE TITLE
Get credentials accessibility labels missing or misaligned 

### ISSUE THREAD

**Issue by jleach** [bcgov/bc-wallet-mobile#1540]
### Description of problem

The labels on the get credentials action sheet are missing or not aligned with the related component.

### Screenshots and/or log output

![IMG_0087](https://github.com/bcgov/bc-wallet-mobile/assets/390891/75d62588-c994-4060-8542-f7b3801ca937)

### Environment
- [ ] Occurs on Android
- [x] Occurs on iOS

Build #:

1252

Android Device Model: 

iOS Device Model: 

### Severity
- [ ] High
- [ ] Medium
- [ ] Low
